src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -16,49 +21,216 @@
 public class ClimbSubsystem extends SubsystemBase {
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
+  private NetworkTableEntry sbclimbHeightOne;
+  private NetworkTableEntry sbclimbHeightTwo;
+  private NetworkTableEntry sbClimbEnabbled;
   private Joystick m_climbJoystick;
   private MotorController m_climbMotorControllerOne;
   private MotorController m_climbMotorControllerTwo;
   private DigitalInput m_limitSwitch;
-  private boolean toggleClimb;
-  private double climbHeight;
-
-  // TODO: maybe add servos
+  private boolean climbEnabbled;
+  private double climbHeightOne;
+  private double climbHeightTwo;
+  private double joystickAxis;
+  private ShuffleboardTab climberTab;
+  private NetworkTableEntry sbclimberpositionOne;
+  private NetworkTableEntry sbclimberspeedOne;
+  private NetworkTableEntry sbclimberheightOne;
+  private NetworkTableEntry sbclimberpositionTwo;
+  private NetworkTableEntry sbclimberspeedTwo;
+  private NetworkTableEntry sbclimberheightTwo;
+  private NetworkTableEntry sbclimbHeightImputOne;
+  private NetworkTableEntry sbclimbHeightImputTwo;
+  private NetworkTableEntry sbClimberSpeedInput;
+  private NetworkTableEntry sbClimbingMode;
 
   public ClimbSubsystem(Joystick joystick) {
     m_climbJoystick = joystick;
-    toggleClimb = false;
-    climbHeight = 0;
+    climbEnabbled = false;
+    climbHeightOne = 0;
+    climbHeightTwo = 0;
 
     // One is left, two is right
+
     m_climbMotorControllerOne =
-        new MotorController("Climb Motor One", Constants.ClimbMotorOne, 40, true);
+        new MotorController("Climb Motor One", Constants.ClimbMotorOne, 60, true);
     m_climbMotorControllerTwo =
-        new MotorController("Climb Motor Two", Constants.ClimbMotorTwo, 40, true);
+        new MotorController("Climb Motor Two", Constants.ClimbMotorTwo, 60, true);
     m_climbMotorControllerTwo.setInverted(true);
-    m_climbMotorControllerTwo.setFollow(m_climbMotorControllerOne);
+    // m_climbMotorControllerTwo.getPID().setOutputRange(-.4, .4);
+    // m_climbMotorControllerOne.getPID().setOutputRange(-.4, .4);
     m_climbMotorControllerOne.getEncoder().setPosition(0);
+    m_climbMotorControllerTwo.getEncoder().setPosition(0);
+
+    m_climbMotorControllerOne.setIdleMode(IdleMode.kBrake);
+    m_climbMotorControllerTwo.setIdleMode(IdleMode.kBrake);
+
+    // m_limitSwitch = new DigitalInput(Constants.LimitSwitchChannel);
+
+    climberTab = Shuffleboard.getTab("ClimbBase");
+    sbClimbingMode =
+        climberTab
+            .add("Manual Mode Enable", false)
+            .withSize(2, 2)
+            .withPosition(5, 1)
+            .withWidget(BuiltInWidgets.kToggleSwitch)
+            .getEntry();
+
+    sbClimberSpeedInput =
+        climberTab.add("Climber Speed input", 0).withSize(2, 2).withPosition(5, 0).getEntry();
```

Comment:
Set the default to something other than 0.

Reply:
set to 0.1